.github/workflows/build-plugin-archive.yml

```
@@ -202,16 +224,37 @@ jobs:
           composer global require inpsyde/wp-translation-downloader
           composer wp-translation-downloader:download
 
+      - name: Run Rector
+        if: steps.composer-tools.outputs.rector == '0'
+        run: |
+          rector
+          
+      - name: Run PHP-Scoper
+        if: steps.composer-tools.outputs.php-scoper == '0'
+        run: |
+          php-scoper add-prefix --force --output-dir=build
+          composer --working-dir=build dump-autoload -o
+          sed -i "s/'__composer_autoload_files'/\'__composer_autoload_files_${{ github.sha }}'/g" "build/vendor/composer/autoload_real.php"
+      - name: Move code to the `build/` directory
+
```

Sorry, my suggestion placed the new line wrongly.
```suggestion

      - name: Move code to the `build/` directory
```